### ISSUE TITLE
cleveref-like referencing

### ISSUE THREAD

**Issue by krlmlr** [rstudio/bookdown#214]
Would you support a `\@cref()` operator that automatically prepends the type of reference (figure, chapter, ...) and also supports multiple references? For instance, `\@cref(fig:a, fig:b)` becomes "Figures 2.1 and 2.2".

**Comment by yihui** [comment]
Yes, I can support this probably in version 0.3 (after the next CRAN release).

**Comment by TrashBirdEcology** [comment]
This would be amazing.
